scripts/exregional_nonvarcldanl.sh

```diff
@@ -274,6 +274,13 @@ else
   n_iolayouty=$(($IO_LAYOUT_Y))
 fi
 
+if [ ${cldclearonly} == "TRUE" ]; then
+  cld_bld_hgt=0.0
+  l_precip_clear_only=.true.
+else
+  cld_bld_hgt=1200.0
+  l_precip_clear_only=.false.
+fi
```

Comment:
Can we put both cld_bld_hgt and l_precip_clear_only into config.sh directly in a similar way as lots of namelist parameters for gsiparm.anl?

Reply:
Good point! This is more straightforward. Changes made. Thank you!